### DISCUSSION TITLE
Feature Ideas!

### DISCUSSION THREAD

**Discussion by chadmcmassive** [Peppermint-Lab/peppermint#167] [Ideas]
I love this service and it is very cool and useful to implement.
Here are a few features I would love to see:

Asset/device inventory and assignment to tickets
Clients being able to sign up and sign in to make their own tickets

Thanks!

**Comment by potts99** [comment]
Yes to both :) 

the second one should be ready soon (lol), hopefully this week
First one ive always liked the idea of but would need a bit of planning

**Reply by chadmcmassive** [reply]
Hey thanks for the quick reply! That's awesome to hear!

**Reply by potts99** [reply]
Thanks for starting a thread :)

**Comment by Grimsons** [comment]
LDAP or full AD integration would be usefull 😊

**Reply by randormi** [reply]
This would be great, I'd love to have more SSO options available.

**Comment by WaldoJoubert-GH** [comment]
Love what you are doing with this project but wanna ask, any plans for dark mode maybe, sorry I work mostly at night and the amount of white is a bit to much.

**Reply by potts99** [reply]
Actively working on dark mode at the moment as I'm the same, just making sure it's nice everywhere before release

**Comment by Pac0Lux** [comment]
In a world of complexity this is so easy.  Love it.

My few suggestions.
Hide Admin Accounts from Assignment Dropdown, Client dropdown when editing existing tickets or ones received by email (Seems it is only available when Creating A ticket), Mail Notifications on Assignment, and MFA (Compliance Issue for some) is about the only thing that I can think for have to haves.  

Harder things that would be cool is the LDAP integration suggested and maybe Auto Populating on Fields from incoming emails.  You set the requirement for import, and we can adjust our web submission forms.   

But this is great, keep it simple and most of all keep having fun.

**Comment by LucaLikesCoffee** [comment]
Perhaps a quality of life request, but when something is submitted from a text box such as comments, clearing the box for the next comment etc.

This would apply to similar areas such as the text box(s) in ticket creation which as an aside could use some more obvious on page 'Toasts' to highlight that the ticket has been created. 

I know this is still early days but honestly this is pretty awesome so far, keep up the good work!

**Comment by Vova-code** [comment]
Super project !

May i suggest you to add the possibility to delete documents directly from document list, it would be a good idea (with role permission and alert box confirmation of course)

![image](https://github.com/user-attachments/assets/2bb8a124-07a2-40c8-9bf7-83bc1ac5e7e3)

I just discovered your project through [dokploy](https://dokploy.com/) templates
Keep going on your efforts !

**Reply by potts99** [reply]
you should be able to right click here, I'll add an on hover actions button 👍

**Comment by oztech74l** [comment]
Why have I learned about this just now? Great product.

Would love to see a feature option where we could customize the logo to put the company name or something. Thanks!

**Comment by ItsNoted** [comment]
Allow embedding of guest ticket page into existing websites. This would be great!
